daemon/loop.go

```diff
@@ -207,9 +207,9 @@ func (d *Daemon) doSync(logger log.Logger, lastKnownSyncTagRev *string, warnedAb
 
 	var resourceErrors []event.ResourceError
 	if err := fluxsync.Sync(syncSetName, allResources, d.Cluster); err != nil {
-		logger.Log("err", err)
 		switch syncerr := err.(type) {
 		case cluster.SyncError:
+			logger.Log("err", err)
```

Comment:
Not quite sure of the motivation for this change -- what errors were being logged that shouldn't be?

Reply:
The problem was that the error was logged twice, by this function and the caller

Reply:
Got it, ta